Add disclaimer

Malvoz opened the pull request:
@nchan0154 Does this make sense to you? 😄

Also, in general, if you find any spelling/grammatical errors don't hesitate to call me out on it!

nchan0154 commented:
A valuable disclaimer! I think its also worth mentioning that the combination of screen readers/browsers tested is not comprehensive and different combinations may have different results. @Malvoz

Malvoz commented:
> also worth mentioning that the combination of screen readers/browsers tested is not comprehensive and different combinations may have different results

@nchan0154 I added a note in the Focus Order SC, because it is currently the only one with screen reader output data.

I also added a note about an NVDA bug which is visible in the output data as it affects the majority of maps' controls.

Malvoz commented:
I'm merging this, but let me know if you think I should change something. Thanks for the feedback Nic!
